File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        // a ⊕ b와 b ⊕ a를 계산
        int ab = Integer.parseInt(String.valueOf(a) + String.valueOf(b));
        int ba = Integer.parseInt(String.valueOf(b) + String.valueOf(a));
        
        // 두 값 중 더 큰 값을 반환
        return Math.max(ab, ba);
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = 0

    ab = str(a) + str(b)
    ba = str(b) + str(a)

    if int(ab) > int(ba):
        answer = int(ab)
    elif int(ba) > int(ab):
        answer = int(ba)
    elif int(ab) == int(ba):
        answer = int(ab)

    return answer